policy.go

```diff
@@ -5,6 +5,8 @@ import (
 	"github.com/coniks-sys/libmerkleprefixtree-go/internal"
 )
 
+type Epoch int64
```

Comment:
I think naming the field in the policy ``Epoch`` is misleading now. Epochs should be the discreet points at which snapshots are taken.  I think ``timestamp`` or ``deadline`` or something similar makes more sense for what we're indicating in the policy now.